testing/can-messages/src/messages.rs

```diff
@@ -61,6 +61,11 @@ pub struct Foo {
 impl Foo {
     pub const MESSAGE_ID: u32 = 256;
 
+    pub const VOLTAGE_MIN: f32 = 0_f32;
+    pub const VOLTAGE_MAX: f32 = 63.9990234375_f32;
```

Comment:
What to do with those?

Reply:
Looks fine to me -- [dbc source](https://github.com/technocreatives/dbc-codegen/blob/62469fa0a328178f1d30b886886a6111b80ae79e/testing/dbc-examples/example.dbc#L12) has the same value.